src/types.ts

```diff
@@ -161,6 +161,18 @@ export type AssetTransferApiOpts = {
 	registryType?: RegistryTypes;
 };
 
+export type AssetTransferApiInjectedOpts = {
```

Comment:
We dont need to make a copy of the `AssetTransferApiOpts` type just use Generics to pass in a type that's necessary.